### DISCUSSION TITLE
Replacement for Google Fonts

### DISCUSSION THREAD

**Discussion by lclc** [plausible/analytics#1682] [Anything Else]
Latest news:

> [German Court Rules Websites Embedding Google Fonts Violates GDPR](https://thehackernews.com/2022/01/german-court-rules-websites-embedding.html)

Could Plausible provide an alternative?

**Comment by metmarkosaric** [comment]
we have our hands full having to focus on web analytics at the moment so there's no short term plans to go into other niches but it's an interesting idea for sure @lclc!
